File: app/src/main/java/org/metabrainz/android/util/Log.java
package org.metabrainz.android.util;

import org.metabrainz.android.config.Configuration;

public class Log {

    public static final String TAG = Configuration.TAG;

    public static void e(String message) {
        android.util.Log.e(TAG, message);
    }

    public static void d(String message) {
        android.util.Log.d(TAG, message);
    }

    public static void v(String message) {
        android.util.Log.v(TAG, message);
    }

}

File: app/src/main/java/org/metabrainz/android/util/Log.kt
package org.metabrainz.android.util

import android.util.Log

object Log {

    private const val TAG = Configuration.TAG

    fun e(message: String?) {
        Log.e(TAG, message!!)
    }

    fun d(message: String?) {
        Log.d(TAG, message!!)
    }

    fun v(message: String?) {
        Log.v(TAG, message!!)
    }
}